docs/content/en/docs/reference/clusterspec/gitops.md

```diff
@@ -8,7 +8,133 @@ description: >
+In order to add GitOps support, you need to configure your cluster by updating the configuration file before creating the cluster.
+We currently support two types of configurations: `FluxConfig` and `GitOpsConfig`.
+
+## Flux Configuration
+The flux configuration spec has three optional fields, regardless of the chosen git provider.
+
+### Flux Configuration Spec Details
+### __systemNamespace__ (optional)
+* __Description__: Namespace in which to install the gitops components in your cluster. Defaults to `flux-system`
+* __Type__: string
+
+### __clusterConfigPath__ (optional)
+
+* __Description__: The path relative to the root of the git repository where EKS Anywhere will store the cluster configuration files. Defaults to the cluster name
+* __Type__: string
+
+### __branch__ (optional)
+
+* __Description__: The branch to use when committing the configuration. Defaults to `main`
+* __Type__: string
+
+EKS Anywhere currently supports two git providers for FluxConfig: Github and Git.
+
+## Github provider
+Please note that for the Flux config to work successfully with the Github provider, the environment variable `EKSA_GITHUB_TOKEN` needs to be set with a valid [GitHub PAT](https://github.com/settings/tokens/new).
+This is a generic template with detailed descriptions below for reference:
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  ...
+  #GitOps Support
+  gitOpsRef:
+    name: my-github-flux-provider
+    kind: FluxConfig
+---
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: FluxConfig
+metadata:
+  name: my-github-flux-provider
+spec:
+  systemNamespace: "my-alternative-flux-system-namespace"
+  clusterConfigPath: "path-to-my-clusters-config"
+  branch: "main"
+  github:
+    personal: true
+    repository: myClusterGitopsRepo
+    owner: myGithubUsername
+
+---
+```
+
+### github Configuration Spec Details
+### __repository__ (required)
+
+* __Description__: The name of the repository where we will store your cluster configuration, and sync it to the cluster. If the repository exists, we will clone it from the git provider; if it does not exist, we will create it for you.
+* __Type__: string
+
+### __owner__ (required)
+
+* __Description__: The owner of the git repository; either a github username or github organization name. The Personal Access Token used must belong to the owner if this is a personal repository, or have permissions over the organization if this is not a personal repository.
+* __Type__: string
+
+### __personal__ (optional)
+
+* __Description__: Is the repository a personal or organization repository? If personal, this value is true; otherwise, false. If using an organizational repository (e.g. personal is false) the owner field will be used as the organization when authenticating to github.com (http://github.com/)
+* __Default__: true
+* __Type__: boolean
+
+## Git provider
```

Comment:
```suggestion
### Git provider
```
Same as my comment above. I added a # to this heading because I think you mean it to be a subhead of "Flux Configuration".